### PR TITLE
docs: add ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs-cn/pull/1084 
PTAL again. @lilin90 @xiekeyi98 @morgo

### CODE REVIEW

**Comment by yikeke** [comment]
> Because this is a English doc repo, shall we delete all the Chinese?

Makes sense, I'll delete them.

File: .github/ISSUE_TEMPLATE/error-report.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+---
+
+## Error Report
+
+**This repository is ONLY used to solve issues related to DOCS.
+For other issues (related to TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**
+
+Please answer the following questions before submitting your issue. Thanks!
+
+1. What is the directory of the document related to this issue?
```
Comment (morgo) [inline .github/ISSUE_TEMPLATE/error-report.md]: Do we want to know the directory, or a url/path?

Comment (yikeke) [reply .github/ISSUE_TEMPLATE/error-report.md]: Yes, the URL/path of the doc is what we want more. I'll change it.

**Comment by morgo** [comment]
LGTM

File: .github/ISSUE_TEMPLATE/change-request.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+name: "\U0001F680 Change Request"
+about: Any suggestion? Want to add something new?
+---
+
+## Change Request
+
+**This repository is ONLY used to solve issues related to DOCS.
+For other issues (related to TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**
```
Comment (lilin90) [inline .github/ISSUE_TEMPLATE/change-request.md]:
```suggestion
For other issues (related to TiDB), please move to [other repositories](https://github.com/pingcap/).**
```

Comment (lilin90) [reply .github/ISSUE_TEMPLATE/change-request.md]: All TiKV related repositories are now in a separate place [here](https://github.com/tikv).

File: .github/ISSUE_TEMPLATE/error-report.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+name: "\U0001F41B Error Report"
+about: Typos, grammatical errors, terminology misuse, ambiguity, etc.
+---
+
+## Error Report
+
+**This repository is ONLY used to solve issues related to DOCS.
+For other issues (related to TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**
```
Comment (lilin90) [inline .github/ISSUE_TEMPLATE/error-report.md]: ditto

File: .github/ISSUE_TEMPLATE/question.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+name: "\U0001F914 Question"
+about: Usage question that isn't answered in docs or discussion
+---
+
+## Question
+
+**This repository is ONLY used to solve issues related to DOCS.
+For other issues (related to TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**
```
Comment (lilin90) [inline .github/ISSUE_TEMPLATE/question.md]: ditto

**Pull request merged**